make build fails :(

```
><> make build
go build -o rootfs/bin/boot -a -installsuffix cgo -ldflags "-s -X main.version=0.0.1-20151105131812"  boot.go
vendor/github.com/deis/pkg/etcd/members.go:10:2: cannot find package "github.com/deis/deis/pkg/k8s" in any of:
        /home/bacongobbler/go/src/github.com/deis/etcd/vendor/github.com/deis/deis/pkg/k8s (vendor tree)
        /usr/local/go/src/github.com/deis/deis/pkg/k8s (from $GOROOT)
        /home/bacongobbler/go/src/github.com/deis/deis/pkg/k8s (from $GOPATH)
Makefile:33: recipe for target 'build' failed
make: *** [build] Error 1
```

Looks like we're missing a dep in glide.yaml

--- a/etcd/members.go
+++ b/etcd/members.go
@@ -7,7 +7,7 @@ import (
 	"github.com/Masterminds/cookoo"
 	"github.com/Masterminds/cookoo/log"
 	"github.com/coreos/etcd/client"
-	"github.com/deis/deis/pkg/k8s"
+	"github.com/deis/pkg/k8s"
 
 	"k8s.io/kubernetes/pkg/labels"
 )

